src/validation.cpp

```diff
@@ -4728,15 +4792,14 @@ bool CChainState::AcceptBlock(const std::shared_ptr<const CBlock>& pblock, CVali
             return state.DoS(100, error("%s: Blockheader marked as PoS but block is not PoS", __func__));
         
         uint256 hashProofOfStake = uint256();
-        std::unique_ptr<CStakeInput> stake;
-
+        std::unique_ptr<StakeInput> stake;
```

Comment:
Shouldn't this stay as `CStakeInput` as its a Class?

Reply:
done